packages/shared/src/view/__tests__/AdaptableLayout.wide.test.js

```diff
@@ -12,28 +12,21 @@ jest.mock('../../Device', () => ({
 
 const RendersCorrectly = () => 'I am going to render without any problems...';
 
-const ThrowsError = () => {
-  throw new Error('do not render');
-};
-
 it('renders wide components', () => {
   expect(
     Renderer.create(
       <Dimensions.Provider dimensions={{ width: 700, height: 500 }}>
-        <AdaptableLayout
-          renderOnWide={<RendersCorrectly />}
-          renderOnNarrow={<ThrowsError />}
-        />
+        <AdaptableLayout renderOnWide={<RendersCorrectly />} />
       </Dimensions.Provider>,
     ),
   ).toMatchSnapshot();
 });
 
-it('renders empty element if there is no wide layout', () => {
+it('renders narrow component if there is no wide one', () => {
   expect(
     Renderer.create(
       <Dimensions.Provider dimensions={{ width: 200, height: 500 }}>
-        <AdaptableLayout renderOnNarrow={<ThrowsError />} />
+        <AdaptableLayout renderOnNarrow={<RendersCorrectly />} />
```

Comment:
How is this related to RN 57? This is obviously wrong - see the generated snapshot: `RendersCorrectly` is not rendered at all and therefore this test doesn't work anymore. It was correct with `ThrowsError` component, or not? (the same for the first test) cc @tbergq

Reply:
Thanks, I missed that detail